### DISCUSSION TITLE
life

### DISCUSSION THREAD

**Discussion by Redmajik** [community/community#141215] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I'm new to it all :) but over a year ago maybe two I have been getting hacked by my neighbor hood, they have my IP address connected to 12 come to find out the modem is a community for them to nest off of my solar panels my internets and resources, I just found out about citations and being suspended and investigated ..question is how do I fix this and make it right?

**Comment by ebndev** [comment]
Hi @Redmajik, thanks for reaching out! 

We're sorry to hear you're experiencing this issue.

GitHub is a software hosting service, we host millions of projects, so probably every phone and computer on the planet has software on it that is also hosted here.

We're not able to provide support or answers for those individual projects. GitHub Support can only provide support for GitHub itself.

I'm sorry we can't help here but we'd recommend contacting a local IT specialist for assistance if you think your devices are affected by viruses or malware. They should be able to help you with these applications!
